tidy3d/components/eme/simulation.py

```diff
@@ -520,17 +524,24 @@ def mode_solver_monitors(self) -> List[ModeSolverMonitor]:
     @property
     def port_modes_monitor(self) -> EMEModeSolverMonitor:
         """EME Mode solver monitor for only the port modes."""
+        if self._sweep_modes:
```

Comment:
how does this fit into `self._num_sweep`? seems like you can just use it directly?